exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import os
+import re
+import shutil
+import subprocess
+import time
+
+from inspect import cleandoc
+from pathlib import Path
+
+
+def arg_parser():
+    parser = argparse.ArgumentParser(
+        description="entry point for docker container",
+    )
+    parser.add_argument(
+        "--copy-from", action="append", type=Path, metavar="<SRC>",
+        help="""Copy files recursively with permissions.
+        For each option --copy-from there must be corresponding option --copy-to.""",
+    )
+    parser.add_argument(
+        "--copy-to", action="append", type=Path, metavar="<DEST>",
+        help="destination location for file to copy",
+    )
+    parser.add_argument(
+        "--jupyter-server", action="store_true",
+        help="start server for Jupyter notebooks",
+    )
+    return parser
+
+
+def start_jupyter_server():
+    subprocess.run([
+        "/root/jupyterenv/bin/jupyter-lab",
```

Comment:
The place of the jupyterenv needs to go into the ansible facts and probably set as ENV in during docker commit and read here from the environment variables.